.circleci/config.yml

```diff
@@ -0,0 +1,90 @@
+version: 2.1
+
+
+orbs:
+  localstack: localstack/platform@2.1
+
+
+jobs:
+  save-state:
+    executor: localstack/default
+    steps:
+      - restore_cache:
+          key: python-deps-
```

Comment:
q: Can this really ever hit on the cache? Doesn't it have to be an exact hit on the key? I would have expected something like `key: python-deps-<hash-of-requirements-file>`...

Reply:
I took the inspiration from [here](https://circleci.com/docs/caching/#source-caching) which if I understand right would use the latest cache from the repo with the given key. So basically the cache don't need to be good initially but during run CircleCI uses the latest available cache with this key, hence our cache during the run is most probably valid.

Reply:
So be it let's do it by the book.